src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Test/ScenarioTests/AsTests.cs

```diff
@@ -37,5 +37,11 @@ public void TestNegativeAnalysisServicesServer()
         {
             NewInstance.RunPsTest(string.Format("Test-NegativeAnalysisServicesServer -location '{0}'", AsTestsBase.resourceGroupLocation));
         }
+
+        [Fact]
+        public void TestAnalysisServicesServerRestart()
+        {
+            NewInstance.RunPsTest(string.Format("Test-AnalysisServicesServerRestart -environment '{0}'", "aspaaswestusloop1.asazure-int.windows.net"));
```

Comment:
Is there a recording for this test?  Also, do not encode internal endppoints, or particular regiosn in test code.